user/vpn.rst

```diff
@@ -0,0 +1,190 @@
+    and `OpenVPN <https://github.com/Stouts/Stouts.openvpn>`_
+
+
+3. Run the Playbook
+~~~~~~~~~~~~~~~~~~~
+
+Run the ansible playbok using:
+
+.. code-block::  bash
+
+    ansible-playbook -i hosts playbook.yml -b -k -K --become-method=su
+
+
+4. Importing the CA and the Server Ceritficate
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To import the CA and Server Ceritficate, you need to access your server
+via ``ssh`` or any other method that suits you.
+
+You need to change your directory to ``/etc/easyrsa/pki/``
+
+.. note::
+    If you face ``-bash: cd: /etc/easyrsa/pki: Permission denied``
+    you may need to login as root user.
+
+**Importing the CA**:
+
+On your openwisp dashboard go to ``/admin/pki/ca/add/``
+
+In **Operation Type** choose :guilabel:`Import Existing`
+
+Get your CA Ceritficate from ``ca.crt`` file and Private Key from
+``private/ca.key`` and then enter them in the respective fields.
+
+**Importing the Server Certificate**:
+
+On your openwisp dashboard go to ``/admin/pki/cert/add/``
+
+In **Operation Type** choose :guilabel:`Import Existing` and in **CA**
+choose the CA you just created.
+
+Get your Server Ceritficate from ``issued/server.crt`` file and Server
+Private Key from ``private/server.key`` and then enter them in the
+respective fields.
+
+4. Creating VPN Server
+~~~~~~~~~~~~~~~~~~~~~~
+
+On your openwisp dashboard go to ``/admin/config/vpn/add/``
+
+In **Host** enter you Server IP, in **Certification Authority** select
+the CA you created and in **X509 Certificate** select the certificate you
+created.
+
+Now under **Configuration**, open **Configuration Menu** and deselect
+Property :guilabel:`Files`. For **VPN1** Change
+:guilabel:`Server (Bridged)` to the Type of your VPN Server. The VPN
+Server installed using the guide above is a Routed Server so change the
+Type to :guilabel:`Server (Routed)`. The Process to setup a Bridged Server
+is identical to that of Routed Server.
+
+Change the rest of the Configuration of the VPN according to the
+configuration in ``/etc/openvpn/server.conf``
+
+.. Tip::
+    You can check if your VPN Configuration is similar to the
+    ``server.conf`` file using the **Preview Configuration** option
+    at the Top.
+
+VPN Templates
+-------------
+
+Creating a VPN Template
+~~~~~~~~~~~~~~~~~~~~~~~
+
+On your openwisp dashboard go to ``/admin/config/template/add/``
+
+Change **Type** to :guilabel:`VPN-client`.
+For **VPN** select the VPN you created.
+
+You can further toggle `Enabled by default <#default-templates>`_
+and `Auto certificate <#auto-client-certificates>`_
+options according to your needs.
+
+Save the template. You can now tweak the Client VPN configuration.
+Now can add the template to your devices.
+
+Auto Client Certificates
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Option**: ``Auto certificate``
+
+**Default**: ``True``
+
+Auto Client Certificates feature allows you to automatically generate
+generate client certificates for your Device.
```

Comment:
Replace `Auto Client Certificates feature allows you` with `This feature allows`.

`for your Device` > `for devices which have this template assigned.`